release/.server.config

```diff
@@ -0,0 +1,17 @@
+#
+# Template configuration file for server environment
+# used by the release shell scripts.
+#
+
+# Base PATH where VCS checkouts are located
+# Code is located in version specific subdirectories named 'squid-*'
+SQUID_VCS_PATH="/var/local/vcs"
+
+# Base PATH where WWW server files are located
+SQUID_WWW_PATH="/var/www"
+
+# Base PATH where rsync shares are located
+SQUID_RSYNC_PATH="/var/rsync"
+
+# Hostname of server performing CI tasks
+SQUID_CI_SERVER="example.com.invalid"
```

Comment:
CI is too general IMO -- we have at least three CI servers already, all performing "CI tasks" (naturally). I do not know which "CI server" this variable defines, so I cannot suggest specific changes, but please detail that in the comment/description and then rename the variable (e.g., to SQUID_BUILD_SERVER or something more suitable).

Reply:
I am not sure either. It is not the name of a host doing the build. It is the name of the CI server coordinating the CI activities and storing the results of those. Thus the name is has now.

I would like to shelve this change request as "good enough" so I can move forward with importing the rest of the automation into this VCS. Followup improvements are welcome of course.

Reply:
> It is the name of the CI server coordinating the CI activities and storing the results of those.

Adding that information to the comment would be slightly better than doing nothing, but I recommend to at least mention an example of a "CI activity" and an example of a "result". Otherwise, there are still too many CI servers that match the description. I know of several Squid CI servers, and I still do not know which one this variable is supposed to name. In fact, every one of them matches the description, for some reasonable definitions of "coordinating the CI activities" and "storing the results of CI activities".

AFAICT, the official squid-ci repository does not use this variable, and the PR itself does not use this variable, so I cannot easily guess what it is based on use cases either.

Going forward:

* If this problematic variable is currently unused in the official repository and this PR, and you want to move forward without fixing its problems, then do not add this variable in this PR.

* If you want to fix its problems, then consider sharing a specific example of this variable (or equivalent) use, current or future, and I will try to suggest a specific fix.

Reply:
Dropped for now.

Reply:
Dropped this for now in [348f28b](https://github.com/squid-cache/ci/pull/7/commits/348f28b49b493d1005d121a6a4e51259533239d6)